services/exec.js

```diff
@@ -0,0 +1,27 @@
+
+'use strict';
+
+const httpExec = require('./httpExec'),
+  ipcExec = require('./ipcExec');
+
+module.exports = async (method, params) => {
+
+  let result = await Promise.all([
```

Comment:
what the hell?, why are you trying to perform multiple requests to different endpoints?, if we have only one active